edsnlp/pipelines/core/matcher/matcher.py

```diff
@@ -35,6 +35,11 @@ class GenericMatcher(BaseComponent):
     ignore_excluded : bool
         Whether to skip excluded tokens (requires an upstream
         pipeline to mark excluded tokens).
+    ignore_space_tokens: bool
```

Comment:
We could add a check on the normalizer pipeline if `ignore_space_token` is set to True ?
Maybe throw a warning if `nlp.get_pipe("eds.normalizer").spaces == None` ?

Reply:
Great idea. To avoid relying on factory names or component names, an even better solution would be to use the `assigns` / `requires` attributes but I doubt we can dynamically define the `requires` depending on the factory parameters. Another issue is that we can only check that such a pipeline, either `eds.normalizer` with spaces or `eds.spaces` from a component factory when `nlp` is given. Would you like to make a PR for this ?